src/integration-tests/config.js/responseSchema.js

```diff
@@ -0,0 +1,95 @@
+const listSchema = {
```

Comment:
Can we add negative cases schema as well? for example here in this case if we attempt to call this api without token , api should throw 401 Unauthorised. Such cases can we handle? @priyanka-TL

Reply:
@adithyadinesh0412  as Akash said we can add negative test cases while developing